cfpb/cfgov-refresh
# Moving owning-a-home/mortage-estimage into cfgov-refresh

## Changes

- Modified build processes, CSS files, JS files, and test files to move mortgage estimate into cfgov-refresh. 

## Testing

1. Run gulp build.
2 Run gulp test:unit and verify that all tests pass.
3. Visit ` http://localhost:8000/owning-a-home/mortgage-estimate/` and compare to `https://www.consumerfinance.gov/owning-a-home/mortgage-estimate/`.


## Notes

- This page is needed to add Nudgy guy to our repo.
- I had to add it's own email sign-up because of the ` {% include_block form %}` tag in our atomic `email-signup.html`. 
- The template is wider because it's using our base.html, as opposed to the base.html template in the oah repo. It now matches the rest of the owning-a-home Wagtail pages. 


## Todos

-

## Checklist
